code/modules/mob/living/carbon/stripping.dm

```diff
@@ -248,6 +249,8 @@
 
 // Modify the current target sensor level.
 /mob/living/carbon/human/proc/toggle_sensors(var/mob/living/user)
+	var/pickpocket = user.isGoodPickpocket()
+	var/delay = HUMAN_STRIP_DELAY / (pickpocket ? 2 : 1)
```

Comment:
You can move these two lines to `strip_time()` instead of copy pasting it two times. Then this line could be `var/delay = user.strip_time()`. I still don't like the idea of calculating the strip delay every time but that's how it currently works for carbons. Maybe we should store it in a variable.
I'd go even further and rename `strip_time` to `get_strip_time()` but that's just me.

Also you really should move this after the suit and sensors checks so you don't process shit you aren't using